olive/engine/packaging/packaging_generator.py

```diff
@@ -67,6 +69,28 @@ def _generate_zipfile_output(
         shutil.move(f"{packaging_config.name}.zip", output_dir / f"{packaging_config.name}.zip")
 
 
+def _package_models_rank(tempdir, footprints: Dict["AcceleratorSpec", "Footprint"]):
+    metrics_dict = next(iter(footprints.values())).objective_dict
+    sorted_nodes = sorted(
+        itertools.chain.from_iterable(f.nodes.values() for f in footprints.values()),
+        key=lambda x: tuple(
+            x.metrics.value[metric].value if x.metrics.cmp_direction[metric] == 1 else -x.metrics.value[metric].value
+            for metric in metrics_dict
+        ),
+        reverse=True,
+    )
+    with (tempdir / "models_rank.json").open("w") as f:
+        rank = 1
+        for node in sorted_nodes:
```

Comment:
is it better to write once if we put all these model info into a list?

Reply:
when I try to load the following json file, the following errors will be raised.
![image](https://github.com/microsoft/Olive/assets/817030/c95eb7fa-788d-45dd-8346-b6ba9ae9cdb2)


```json
{
    "a": "b"
}
{
    "a": "b"
}
{
    "a": "b"
}
```

Reply:
Oh cause I don't expect user to check this file in their code, but they may want to check this file manually. So this file is not a valid json file. But you are right they may want to load this file in their code. I can update this.